packages/lesswrong/components/comments/SingleLineComment.tsx

```diff
@@ -187,7 +187,7 @@ const SingleLineComment = ({treeOptions, comment, nestingLevel, parentCommentId,
         {!hideKarma && <span className={classes.karma}>
           {commentGetKarma(comment)}
         </span>}
-        <CommentUserName comment={comment} simple={true} className={classes.username} />
+        <CommentUserName comment={comment} simple={true} hideSprout className={classes.username} />
```

Comment:
I figured it's not worth displaying the icon when the comment is so truncated that you can't really read it anyway